cfgov/cfgov/settings/base.py

```diff
@@ -720,6 +720,13 @@
 
     # Used to hide CCDB landing page updates prior to public launch.
     'CCDB_CONTENT_UPDATES': [],
+
+    # During a Salesforce system outage, the following flag should be enabled
+    # to alert users that the Collect community is down
+    'COLLECT_OUTAGE': [
+        {'condition': 'boolean', 'value': False},
+        {'condition': 'path matches', 'value': r'^/data-research/credit-card-data/terms-credit-card-plans-survey|^/data-research/prepaid-accounts', 'required': True}
```

Comment:
Not really possible to elegantly avoid exceeding the line limit (hence the `noqa` directive), but this will at least make it a little more readable:

```suggestion
        {
            'condition': 'path matches',
            'value': (r'^/data-research/credit-card-data/terms-credit-card-plans-survey|'  # noqa: E501
                      r'^/data-research/prepaid-accounts'),
            'required': True
        }
```